extension/src/popup/components/PasswordConfirmation.tsx

```diff
@@ -0,0 +1,99 @@
+import React from "react";
```

Comment:
this is simply taking the already existing Password Confirmation screen and turning it into a reusable component as we will need the user's password to add an account